dao/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -34,25 +34,20 @@
     <include file="db/changelog/logs/ch-eco-news-Kovaliv-4.xml"/>
     <include file="db/changelog/logs/ch-change-users-addColumn-datsko.xml"/>
     <include file="db/changelog/logs/ch-change-fact-translations-Patsula.xml"/>
-    <include file="db/changelog/logs/ch-add-tips-and-tricks-Ostriychuk-1.xml"/>
     <include file="db/changelog/logs/ch-add-econews-comment-Patsula.xml"/>
     <include file="db/changelog/logs/ch-users-friends-Datsko.xml"/>
     <include file="db/changelog/logs/ch-change-users-rating-Datsko.xml"/>
     <include file="db/changelog/logs/ch-change-eco-news-Ostriychuk.xml"/>
     <include file="db/changelog/logs/ch-add-column-Zhurakovskyi-1.xml"/>
-    <include file="db/changelog/logs/ch-add-tipsandtricks-comment-Beshlei.xml"/>
     <include file="db/changelog/logs/ch-user-profile-Datsko.xml.xml"/>
-    <include file="db/changelog/logs/ch-merge-tags-Ostriychuk.xml"/>
     <include file="db/changelog/logs/ch-habit-status-Beshlei-2.xml"/>
     <include file="db/changelog/logs/ch-habits-Beshlei-1.xml"/>
     <include file="db/changelog/logs/ch-habit-status-calendar-Beshlei-1.xml"/>
     <include file="db/changelog/logs/ch-primarykey-eco-news-tags-Lehkyi-1.xml"/>
-    <include file="db/changelog/logs/ch-primarykey-tips-and-tricks-tags-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch-fact-of-the-day-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch-social-network-Lehkyi.xml"/>
     <include file="db/changelog/logs/ch_rating_statictics_Dovganyuk.xml"/>
     <include file="/db/changelog/logs/ch-user-username-column-type.xml"/>
-    <include file="/db/changelog/logs/ch-tips-and-tricks-translations-Petryshak.xml"/>
```

Comment:
Don't remove the change set, just write new, that will delete the table or column that you need to delete

Reply:
agree. create separate changeset that will cleanup unused tables/columns

Reply:
Done